.github/workflows/build.yml

```diff
@@ -26,7 +26,7 @@ jobs:
            complex: --complex
     env:
       FIREDRAKE_CI_TESTS: 1
-      PETSC_CONFIGURE_OPTIONS: --with-make-np=12 --download-mpich-device=ch3:sock
+      PETSC_CONFIGURE_OPTIONS: --with-make-np=12 --download-mpich-device=ch3:sock --download-cmake
```

Comment:
If this is needed on the build system, is it also needed on the current debian-based platforms?

Reply:
It is needed only when the system cmake is older than 3.18.1; added a comment.

Reply:
what I mean is, if we need to do this on the ubuntu:20.04 images we’re using for the builders, do we need to do it by default in firedrake-install for most apt-based systems?

Reply:
Thanks. Fixed now.